pkg/apis/messaging/v1alpha1/channel_defaults.go

```diff
@@ -19,10 +19,10 @@ package v1alpha1
 import (
 	"context"
 
+	"github.com/google/knative-gcp/pkg/logging"
```

Comment:
did we move the eventing logging here?
Probably the eventing logging was removing and they are now using the pkg logging in eventing. 
If that is the case, we should do the same here

Reply:
pkg logging and eventing logging have different signatures: eventing logging returns a Logger, pkg logging returns a SugaredLogger. Most of the code in this repo expects a Logger.

Reply:
ack, fine to keep it then